pkg/widgets/panel.go

```diff
@@ -139,7 +139,9 @@ func (p *Panel) SetLocation(x0, y0, x1, y1 int) {
 }
 
 func (p *Panel) SetContent(content string) {
-	panelWidth := p.X1 - p.X0
+	// Need to subtract 1 from panelWidth here to match the view.Size()
+	// calculation done in gocui's view.Draw() function
+	panelWidth := p.X1 - p.X0 - 1
```

Comment:
Is there any chance at all of this becoming negative? Would that be bad or does gocui do the right thing and just occlude anything that's not on the page?

Reply:
I don't believe it's possible for that to ever end up negative, but just in case I'm wrong there's an `if panelWidth > 0` to ensure our wrapping doesn't kick in if the dimensions are bogus :-)